docs/cvl/hooks.md

```diff
@@ -46,6 +46,13 @@ See {doc}`statements` for information about the `statement` production; see
 {doc}`types` for the `evm_type` production; see {doc}`basics` for the `number`
 production.
 
+It is generally prohibited to hook more than once on any specific opcode or
+storage location to avoid ambiguities on the order of hook applications.
```

Comment:
This is very academic, so maybe add some simple sentences like: "The typechecker will emit a suitable error if multiple hooks on the same pattern are defined"

Reply:
I rewrote the text to be simpler (not sure how well that worked out...)
I also realized that my claim that access path resolution is injective (and thus access paths are different if they are syntactically different) is not true.